src/app/shapes/Triangle.ts

```diff
@@ -16,6 +16,11 @@ export class Triangle extends Shape {
     public getTop() : number{
         return Math.min(this.instance.points()[1], this.instance.points()[3], this.instance.points()[5]);
     }
+
+    public move(x: number, y: number): void {
+        this.instance.x(x);
+        this.instance.y(y);
+    }
    
```

Comment:
I'm actually not sure that this will work since we never set x and y initially. I think you would have to modify the x1, y1, x2, y2, x3, y3 points individually based on the initial or we need to initialize x and y to something. I'm not sure though.

Reply:
So after I left I realized this won't work but for sort of an unexpected reason. This will move the triangle on the screen even though we don't initialize x and y (Konva must do that for us). However, this will break the getLeft() and getTop() methods because moving the shape doesn't change the points array which is what we use for those methods. I'll be working on a solution to this problem!